opentelemetry-sdk/src/opentelemetry/sdk/_logs/_internal/export/__init__.py

```diff
@@ -0,0 +1,330 @@
+# Copyright The OpenTelemetry Authors
```

Comment:
Why doesn't github realize this was a rename? Makes it hard to tell if any code was edited in this file, ill assume not